include/mqtt/endpoint.hpp

```diff
@@ -6379,7 +6367,7 @@ class endpoint : public std::enable_shared_from_this<endpoint<Socket, Mutex, Loc
      *        The ownership of  the packet_id moves to the library.
      * @param topic_name
      *        A topic name to subscribe
-     * @param option
+     * @param options
```

Comment:
`options` is still remained. Maybe it is the last one.

Reply:
Any left?